deployment/CP04OSSM_Deploy.csv

```diff
@@ -86,86 +86,86 @@ AT-31,,AR-04,,CP04OSSM-RID27-03-CTDBPC000,3,1,2015-10-22T19:38:00,2016-05-13T17:
+AR-08,,AR-18,,CP04OSSM-RIC21-00-CPMENG000,5,1,2016-10-12T14:26:00,2017-06-08T12:05:00,CGMCP-04OSSM-00005,,CGCON-NCPM02-00005,39.93707,-70.88697,,7,450,
+AR-08,,AR-18,,CP04OSSM-RID26-00-DCLENG000,5,1,2016-10-12T14:26:00,2017-06-08T12:05:00,CGMCP-04OSSM-00005,,CGCON-NDCL26-00024,39.93707,-70.88697,,7,450,
+AR-08,,AR-18,,CP04OSSM-RID26-04-VELPTA000,5,1,2016-10-12T14:26:00,2017-06-08T12:05:00,CGMCP-04OSSM-00005,,CGINS-VELPTA-12711,39.93707,-70.88697,,7,450,
+AR-08,,AR-18,,CP04OSSM-RID26-06-PHSEND000,5,1,2016-10-12T14:26:00,2017-06-08T12:05:00,CGMCP-04OSSM-00005,,CGINS-PHSEND-P0087,39.93707,-70.88697,,7,450,
+AR-08,,AR-18,,CP04OSSM-RID26-07-NUTNRB000,5,1,2016-10-12T14:26:00,2017-06-08T12:05:00,CGMCP-04OSSM-00005,,CGINS-NUTNRB-00260,39.93707,-70.88697,,7,450,
+AR-08,,AR-18,,CP04OSSM-RID26-08-SPKIRB000,5,1,2016-10-12T14:26:00,2017-06-08T12:05:00,CGMCP-04OSSM-00005,,CGINS-SPKIRB-00289,39.93707,-70.88697,,7,450,
+AR-08,,AR-18,,CP04OSSM-RID27-00-DCLENG000,5,1,2016-10-12T14:26:00,2017-06-08T12:05:00,CGMCP-04OSSM-00005,,CGCON-NDCL27-00005,39.93707,-70.88697,,7,450,
+AR-08,,AR-18,,CP04OSSM-RID27-01-OPTAAD000,5,1,2016-10-12T14:26:00,2017-06-08T12:05:00,CGMCP-04OSSM-00005,,CGINS-OPTAAD-00153,39.93707,-70.88697,,7,450,
+AR-08,,AR-18,,CP04OSSM-RID27-02-FLORTD000,5,1,2016-10-12T14:26:00,2017-06-08T12:05:00,CGMCP-04OSSM-00005,,CGINS-FLORTD-01323,39.93707,-70.88697,,7,450,
+AR-08,,AR-18,,CP04OSSM-RID27-03-CTDBPC000,5,1,2016-10-12T14:26:00,2017-06-08T12:05:00,CGMCP-04OSSM-00005,,CGINS-CTDBPC-50056,39.93707,-70.88697,,7,450,
+AR-08,,AR-18,,CP04OSSM-RID27-04-DOSTAD000,5,1,2016-10-12T14:26:00,2017-06-08T12:05:00,CGMCP-04OSSM-00005,,CGINS-DOSTAD-00507,39.93707,-70.88697,,7,450,
+AR-08,,AR-18,,CP04OSSM-SBC11-00-CPMENG000,5,1,2016-10-12T14:26:00,2017-06-08T12:05:00,CGMCP-04OSSM-00005,,CGCON-BCPM01-00007,39.93707,-70.88697,,0,450,
+AR-08,,AR-18,,CP04OSSM-SBD11-00-DCLENG000,5,1,2016-10-12T14:26:00,2017-06-08T12:05:00,CGMCP-04OSSM-00005,,CGCON-BDCL11-00007,39.93707,-70.88697,,0,450,
+AR-08,,AR-18,,CP04OSSM-SBD11-01-MOPAK0000,5,1,2016-10-12T14:26:00,2017-06-08T12:05:00,CGMCP-04OSSM-00005,,CGINS-MOPAK0-42298,39.93707,-70.88697,,0,450,
+AR-08,,AR-18,,CP04OSSM-SBD11-02-HYDGN0000,5,1,2016-10-12T14:26:00,2017-06-08T12:05:00,CGMCP-04OSSM-00005,,CGINS-HYDGN0-49857,39.93707,-70.88697,,0,450,
+AR-08,,AR-18,,CP04OSSM-SBD11-06-METBKA000,5,1,2016-10-12T14:26:00,2017-06-08T12:05:00,CGMCP-04OSSM-00005,,CGINS-METLGR-00016,39.93707,-70.88697,,0,450,
+AR-08,,AR-18,,CP04OSSM-SBD12-00-DCLENG000,5,1,2016-10-12T14:26:00,2017-06-08T12:05:00,CGMCP-04OSSM-00005,,CGCON-BDCL12-00007,39.93707,-70.88697,,0,450,
+AR-08,,AR-18,,CP04OSSM-SBD12-03-HYDGN0000,5,1,2016-10-12T14:26:00,2017-06-08T12:05:00,CGMCP-04OSSM-00005,,CGINS-HYDGN0-49886,39.93707,-70.88697,,0,450,
+AR-08,,AR-18,,CP04OSSM-SBD12-04-PCO2AA000,5,1,2016-10-12T14:26:00,2017-06-08T12:05:00,CGMCP-04OSSM-00005,,CGINS-PCO2AA-34223,39.93707,-70.88697,,0,450,
```

Comment:
Reviewed corrections to deployment CUID, sensor UIDs and deployment depths. All look good.